### ISSUE TITLE
Discrepancies from $3 to $14 of EITC AGI maximum against IRS publication

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/openfisca-us#1052]
In #1049 I'm adding the EITC AGI maximum, which is used in the law for the Washington Working Families' Tax Credit. Compared to these [IRS publication](https://www.irs.gov/credits-deductions/individuals/earned-income-tax-credit/earned-income-and-earned-income-tax-credit-eitc-tables), these are generally off by about $4 for 2021. The value for zero children, filing joint is most off: OpenFisca US says it's $27,366.99, and the IRS says it's $27,380. We might be missing some rounding rule.

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/6076111/177665750-273cd116-8a16-4026-80ac-f607f95dd9ae.png">

**Comment by nikhilwoodruff** [comment]
Yeah I'd be surprised if it's a rounding rule, since we're using the other EITC parameters from the official IRS release?

**Comment by DTrim99** [comment]
@MaxGhenis Closing as obsolete — the eitc_agi_limit variable exists; the $3-$14 difference vs the IRS publication is the expected exact-formula-vs-rounded-table gap (confirmed by martinholmer in-thread), not a defect (no recent activity). Reopen if a real discrepancy is found.
